### PR TITLE
non-split-customer-endpoints-tests

### PR DESCRIPTION
**Pull request by shubham-khandelwal2-metacube**
## PR Description

customer-restore-password https://spryker.atlassian.net/browse/CC-16810
customer-password https://spryker.atlassian.net/browse/CC-16809
customer-confirmation https://spryker.atlassian.net/browse/CC-16807
customer-forgotten-password https://spryker.atlassian.net/browse/CC-16808

### CODE REVIEW

File: tests/api/suite_nonsplit/glue/customer_endpoints/customer_confirmation/negative.robot [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    And Response header parameter should be:    Content-Type    ${default_header_content_type}
+
+Customer_confirmation_with_empty_type
+    And I send a POST request:    /customer-confirmation   {"data":{"type":"","attributes":{"registrationKey":"607a17d1c673f461ca40002ea79fddc0"}}}
+    Response status code should be:    400
+    And Response should return error message:    Invalid type.
+    And Response header parameter should be:    Content-Type    ${default_header_content_type}
+
+# need receive the confirmation key from email
+# Customer_confirmation_with_already_used_confirmation_key
```
Comment (OKrivtsova) [inline tests/api/suite_nonsplit/glue/customer_endpoints/customer_confirmation/negative.robot:39]: please see how this test is done in b2c master and apply it there - it should be uncommented
https://github.com/spryker/robotframework-suite-tests/tree/master/tests/api/b2c/glue/customer_endpoints/customer_confirmation

File: tests/api/suite_nonsplit/glue/customer_endpoints/customer_confirmation/positive.robot [hunk 2]
```diff
@@ -0,0 +1,18 @@
+*** Settings ***
+Suite Setup       SuiteSetup
+Resource    ../../../../../../resources/common/common_api.robot
+
+*** Test Cases ***
+# need receive the confirmation key from email
+# Customer_confirmation
```
Comment (OKrivtsova) [inline tests/api/suite_nonsplit/glue/customer_endpoints/customer_confirmation/positive.robot:7]: please see how this test is done in b2c master and apply it there - it should be uncommented
https://github.com/spryker/robotframework-suite-tests/tree/master/tests/api/b2c/glue/customer_endpoints/customer_confirmation

File: tests/api/suite_nonsplit/glue/customer_endpoints/customer_restore_password/positive.robot [hunk 3]
```diff
@@ -0,0 +1,11 @@
+*** Settings ***
+Suite Setup       SuiteSetup
+Resource    ../../../../../../resources/common/common_api.robot
+
+*** Test Cases ***
+# # can't receive the confirmation from email
+# Restore_password_with_all_required_fields_and_valid_data
```
Comment (OKrivtsova) [inline tests/api/suite_nonsplit/glue/customer_endpoints/customer_restore_password/positive.robot:7]: please see how this test is done in b2c master and apply it there - it should be uncommented
https://github.com/spryker/robotframework-suite-tests/blob/master/tests/api/b2c/glue/customer_endpoints/customer_restore_password/positive.robot

**Review by OKrivtsova — COMMENTED** [review]
The changes I asked for can only be checked locally  - on the environment it can be checked only if you have the database credentials.

File: tests/api/suite_nonsplit/glue/customer_endpoints/customer_confirmation/negative.robot (hunk 1 above)
Comment (shubham-khandelwal2-metacube) [reply tests/api/suite_nonsplit/glue/customer_endpoints/customer_confirmation/negative.robot:39]: FIXED

File: tests/api/suite_nonsplit/glue/customer_endpoints/customer_confirmation/positive.robot (hunk 2 above)
Comment (shubham-khandelwal2-metacube) [reply tests/api/suite_nonsplit/glue/customer_endpoints/customer_confirmation/positive.robot:7]: FIXED

File: tests/api/suite_nonsplit/glue/customer_endpoints/customer_restore_password/positive.robot (hunk 3 above)
Comment (shubham-khandelwal2-metacube) [reply tests/api/suite_nonsplit/glue/customer_endpoints/customer_restore_password/positive.robot:7]: FIXED

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**